cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -565,6 +568,104 @@ describe('Data Table', () => {
     });
   });
 
+  describe('Radio selection', () => {
+    beforeEach(() => {
+      cy.get(`[data-cy='data-table-radio']`)
+        .find(`.${DATA_TABLE_CLASSES.SELECTABLE}`)
+        .find(`.${RADIO_CLASSES.RADIO}`)
+        .as('radios');
+    });
+
+    it(`Should have class .${RADIO_CLASSES.RADIO}`, () => {
+      const rows = [1, 2, 3];
+
+      rows.forEach(rowIndex => {
+        cy.get(`[data-cy='data-table-radio'] .${DATA_TABLE_CLASSES.ROW}`)
+          .eq(rowIndex)
+          .find(`.${DATA_TABLE_CLASSES.CELL}`)
+          .eq(0)
+          .find(`.${RADIO_CLASSES.RADIO}`)
+          .as('row');
+        hasClassAssertion(`@row`, RADIO_CLASSES.RADIO);
```

Comment:
```suggestion
          .as('rowRadio');
        hasClassAssertion(`@rowRadio`, RADIO_CLASSES.RADIO);
```

Reply:
Actually, the .find(`.${RADIO_CLASSES.RADIO}`) works using the `RADIO_CLASSES.RADIO`so the assertion will always be true.